Sources/core/Game.cpp

```diff
@@ -153,6 +153,10 @@ void Game::Process()
             ProcessBroadcastedMessages();
             GetFactory().ForeachProcess();
             ForceManager::Get().Process();
+            for(auto& e : projectile_process)
+            {
+                (e.second)->ProcessMovement();
+            }
             if (ATMOS_OFTEN == 1 || MAIN_TICK % ATMOS_OFTEN == 1)
```

Comment:
Same here. Do not implement new global functionality without great needs. You have added some projectile handler, but you have not implemented proper save/load constructions for the handler. Overall, this handler should be removed.